<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>659</width>
    <height>398</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>659</width>
    <height>398</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_General">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>General</string>
   </property>
   <widget class="QCheckBox" name="checkBox_option1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Open URL in Web Browser after 'Generate' button pressed.</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_option2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy URL to clipboard after 'Generate' button pressed.</string>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Keyboard Shortcuts</string>
   </property>
   <widget class="QKeySequenceEdit" name="keySequenceEdit_option1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyboard_shortcuts_option1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Import new playlist</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyboard_shortcuts_option2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Export playlist</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="keySequenceEdit_option2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyboard_shortcuts_option3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear playlist</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="keySequenceEdit_option3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_keyboard_shortcuts_option4">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate playlist</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="keySequenceEdit_option4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>641</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Language</string>
   </property>
   <widget class="QComboBox" name="comboBox_language">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Deutsch</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
